src/org/ods/orchestration/Stage.groovy

```diff
@@ -21,7 +21,7 @@ class Stage {
 
     def execute() {
         script.stage(STAGE_NAME) {
-            script.echo "**** STARTING stage ${STAGE_NAME} ****"
+            script.echo "**** STARTING MRO stage ${STAGE_NAME} ****"
```

Comment:
I think we never talk about `MRO` anymore to the user. Maybe we can replace with `orchestration stage`?

Reply:
ok .. nitpicky you :D